styles/globals.scss

```diff
@@ -20,3 +20,19 @@ a {
 * {
   box-sizing: border-box;
 }
+
```

Comment:
@gdbarnes - I have added the SummaryPanel styles here pending a conversation on what approach we want to take to component css in the project - this ok for now?

Reply:
Yes, fine for now.

I think we should add per component css in the styles dir. (rather than bundled in with the component).

So this code would sit in:
/styles/components/_summary-panel.scss

...which we could then import into the global.scss file.

Happy to discuss 👍

Reply:
Totally agree. I will make that change tomorrow.

Reply:
Done: https://github.com/LBHackney-IT/lbh-housing-ta/pull/2/commits/84bbe04512ef502378fe3220026faf9294ecd9e9